content/_code-samples/escrow/js/get-escrows.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+if (typeof module !== "undefined") {
+  // Use var here because const/let are block-scoped to the if statement.
+  var xrpl = require('xrpl');
+};
+
+const secret = "sEdTPPEeMH6SAgpo6rSj8YW7a9vFfUj";
```

Comment:
Users shouldn't need to enter in their secret since we're just querying the ledger for the account's objects (escrow), the best practice for code that only queries the ledger is to only enter in an address instead of a secret.

Reply:
Removed